### ISSUE TITLE
Lock booking request while in Review

### ISSUE THREAD

**Issue by EvanDime** [CSU-Booking-Platform/application#249]
### User story
As a reviewer, when i start the review process for a booking request, the review will be "locked" and in the state of "in review"
### Narrative
While a booking is being reviewed, we do not want the user booking to be able to update or change anything until the review is complete. 
### Out of scope
Notification of the user about the status of the booking request. 
Make a booking request.
Approve a booking request.
### Acceptance criteria

| # | Given | When | Then |
|---|-------|------|------|
|1|A booking request has been made|A review start reviewing|the status of the booking request changes from "awaiting review" to "in review"|
|2|A user has made a request and it is in review|the user goes to update the booking|it is denied/nothing happens|

**Comment by EvanDime** [comment]
this issue is blocked by #24

**Comment by EvanDime** [comment]
No UI for this as it is going to be the same as #24. This feature is a backend contract
